lib/x509.js

```diff
@@ -2926,7 +2932,7 @@ pki.certificateError = {
  *
  * @return true if successful, error thrown if not.
  */
-pki.verifyCertificateChain = function(caStore, chain, verify) {
+pki.verifyCertificateChain = function(caStore, chain, options) {
```

Comment:
This is the direction we'd go in the future, but it will change the API.  The signature should be `(caStore, chain, verify, options)` for now.  That leaves API open to other options as needed without breaking anything.

Reply:
Ah, I missed that part.  That's fine then.